### ISSUE TITLE
Error inserting CameraCaptureManager_BP blueprint into the scene

### ISSUE THREAD

**Issue by 4ndr3aR** [TimmHess/UnrealImageCapture#33]
Hey there!

First of all let me congratulate you for this plugin. I haven't been able to get it running yet but it looks really well written and should be a great starting point for trying to understand what's going on in the guts of Unreal Engine.

I'm using UE 5.2.0 and I had some trouble adding `CameraCaptureManager_BP.uasset` to the scene. The editor was always giving errors such as:

```
VerifyImport: Failed to find script package for import object 'Package /Script/CameraCaptureToDisk'
CreateExport: Failed to load Parent for BlueprintGeneratedClass /CameraCaptureToDisk/CameraCaptureManager_BP.CameraCaptureManager_BP_C
Unable to load Default__CameraCaptureManager_BP_C with outer Package /CameraCaptureToDisk/CameraCaptureManager_BP because its class (CameraCaptureManager_BP_C) does not exist
```

I tried both using the project already included in your repo and creating new projects (with all kinds of configurations I could think of), without success.

At first I thought what was missing was the reference to the the C++ class, so I tried moving the `CameraCaptureManager.cpp/.h` pair pretty much everywhere, again without success. I finally came across a post about redirects in `Config/DefaultEngine.ini` and that seems to have finally solved my problem.

Basically, after creating a new project with the first person template (called `TestImageCapture1`), I copied `CameraCaptureManager.cpp/.h` to `Source/TestImageCapture1`, `CameraCaptureManager_BP.uasset` to `Content/CameraCaptureToDisk/Blueprints` and I added these two lines to `Config/DefaultEngine.ini` under the `[/Script/Engine.Engine]` section:

```
+ActiveGameNameRedirects=(OldGameName="/Script/CameraCaptureToDisk.CameraCaptureManager",NewGameName="/Script/TestImageCapture1.CameraCaptureManager")
+ActiveClassRedirects=(OldClassName="/Script/CameraCaptureToDisk.CameraCaptureManager",NewClassName="/Script/TestImageCapture1.CameraCaptureManager")
```

Now the blueprint has a parent again and I can add it to the scene.
